frame/glutton/src/lib.rs

```diff
@@ -75,11 +75,20 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event {
 		/// The pallet has been (re)initialized.
-		PalletInitialized { reinit: bool },
+		PalletInitialized {
+			/// The pallet initialization state.
+			reinit: bool,
+		},
 		/// The computation limit has been updated.
-		ComputationLimitSet { compute: FixedU64 },
+		ComputationLimitSet {
+			/// The computation limit.
```

Comment:
Annoying that we say "compute" everywhere else and "computation" here.

Reply:
Yep agree. Right now the goal is to fix the warnings. We may want to do a pass on renaming some the variables to be more accurate in a follow-up. For this one, it makes more sense to use the term "computation" over "compute" imo.